lib/chewy/type.rb

```diff
@@ -41,6 +41,18 @@ def self.type_name
       adapter.type_name
     end
 
+    # Returns the identifier value
+    #
+    def self.identify objects
+      case root_object
```

Comment:
@pyromaniac I think what you're saying is that, this method should be rewritten as 

```ruby
def self.identify objects
  Array(root_object.compose_id(objects) || adapter.identify(objects))
end
```

root_object might always be a Chewy::Fields::Root when the application is actually running, but there are 15 failures that occur if I modify this method to match what I provided in this comment.